prov/efa/src/rdm/efa_rdm_ep_progress.c

```diff
@@ -552,6 +553,7 @@ ssize_t efa_rdm_ep_send_queued_pkts(struct efa_rdm_ep *ep,
 	struct dlist_entry *tmp;
```

Comment:
I think this function should be renamed to `efa_rdm_ep_post_queued_pkts` as it now include both send and write